conf/machine/include/imx-base.inc

```diff
@@ -435,13 +435,13 @@ MACHINE_GSTREAMER_1_0_PLUGIN:mx8qm-nxp-bsp  ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN:mx8qxp-nxp-bsp ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN:mx8dx-nxp-bsp  ?= "imx-gst1.0-plugin"
 
-PREFERRED_VERSION_gstreamer1.0:mx8-nxp-bsp              ??= "1.18.5.imx"
-PREFERRED_VERSION_gstreamer1.0-plugins-base:mx8-nxp-bsp ??= "1.18.5.imx"
-PREFERRED_VERSION_gstreamer1.0-plugins-good:mx8-nxp-bsp ??= "1.18.5.imx"
-PREFERRED_VERSION_gstreamer1.0-plugins-bad:mx8-nxp-bsp  ??= "1.18.5.imx"
-PREFERRED_VERSION_gstreamer1.0-plugins-ugly:mx8-nxp-bsp ??= "1.18.5"
-PREFERRED_VERSION_gstreamer1.0-libav:mx8-nxp-bsp        ??= "1.18.5"
-PREFERRED_VERSION_gstreamer1.0-rtsp-server:mx8-nxp-bsp  ??= "1.18.5"
+PREFERRED_VERSION_gstreamer1.0:mx8-nxp-bsp              ??= "1.20.0.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-base:mx8-nxp-bsp ??= "1.20.0.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-good:mx8-nxp-bsp ??= "1.20.0.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-bad:mx8-nxp-bsp  ??= "1.20.0.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-ugly:mx8-nxp-bsp ??= "1.20.0"
+PREFERRED_VERSION_gstreamer1.0-libav:mx8-nxp-bsp        ??= "1.20.0"
+PREFERRED_VERSION_gstreamer1.0-rtsp-server:mx8-nxp-bsp  ??= "1.20.o"
```

Comment:
1.20.0

Reply:
I'm actually working on this as well.

I guess you are getting a warning for `1.20.0` since it doesn't existing on `master`. I just dropped this preferred version in favor of the current recipes, which are `1.20.3`.

Reply:
@thochstein I just changed, please take a look if I understand it right

Reply:
I thought to just drop the `PREFERRED_VERSION` settings for `1.20.3`, thinking that any future versions `1.20.x` would be compatible. I guess that's been my experience with the first three such versions.

Reply:
@angolini, sorry, I didn't look closely enough at your solution, and I see now that you renamed the downgraded `1.18.5` recipes to `1.20.0`. My thinking is to just drop the `1.18.5` recipes and use the current versions in OE-Core.

I'm ready to submit my changes for a PR...

Reply:
Actually, not quite ready, I remember I still needed to gather the changelogs for the codec/dsp recipes.